backend/src/main/java/me/performancereservation/domain/user/controller/UserController.java

```diff
@@ -0,0 +1,33 @@
+package me.performancereservation.domain.user.controller;
+
+import lombok.RequiredArgsConstructor;
+import me.performancereservation.domain.user.dto.UserMeResponse;
+import me.performancereservation.domain.user.entitiy.User;
+import me.performancereservation.domain.user.service.UserService;
+import me.performancereservation.global.exception.AppException;
+import me.performancereservation.global.exception.ErrorCode;
+import me.performancereservation.global.exception.ErrorType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/v1/user")
+@RequiredArgsConstructor
+public class UserController {
+
+    private final UserService userService;
+    //http://localhost:8080/oauth2/authorization/google 로그인 테스트
+
+    @GetMapping("/me")
+    public ResponseEntity<UserMeResponse> getMyInfo(Authentication authentication) {
+        if (authentication == null || authentication.getPrincipal() == null) {
+            throw new AppException(ErrorCode.UNAUTHORIZED, ErrorType.DOMAIN);
+        }
+        Long userId = (Long) authentication.getPrincipal();
+        User user = userService.getUserById(userId);
```

Comment:
```suggestion
    public ResponseEntity<UserMeResponse> getMyInfo(@AuthenticationPrincipal Authentication authentication) {
```

이렇게 하면 `new CustomOAuth2User(user, oAuth2User.getAttributes());`  이 코드로부터

CustomOAuth2UserService에서 생성한 Authentication 객체에 User가 들어있는 걸로 알고 있어요 바로 사용 가능!

Reply:
@AuthenticationPrincipal 에 대해 공부해보고 필요한 부분 적용시켜놓겠습니당

Reply:
적용 완료했습니다!